osu.Game/Overlays/Chat/ChatTextBar.cs

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics.Containers;
+using osu.Game.Online.Chat;
+using osuTK;
+
+namespace osu.Game.Overlays.Chat
+{
+    public class ChatTextBar : Container
+    {
+        public readonly BindableBool ShowSearch = new BindableBool();
+
+        public event TextBox.OnCommitHandler? OnChatMessageCommit;
```

Comment:
Linking up to the other review comment, ideally I'd like this event to have a signature of `Action<string>?` too - and I wanted to apply this change myself - but the test doing the textbox clearing is not really allowing for that right now.

Also would rename to `OnChatMessageCommitted` for consistency.